cmd/zt_block_size_to_bytes_test.go

```diff
@@ -0,0 +1,54 @@
+// Copyright © 2017 Microsoft <wastore@microsoft.com>
```

Comment:
Minor: just started noticing this, should we change the year to 2019 for new files?

The year says 2017 for the oldest files generated back in December 2017 (with Cobra).

Reply:
It's not clear that updates are strictly necessary. E.g. https://softwareengineering.stackexchange.com/questions/130478/what-copyright-date-for-an-update-to-an-open-source-project-from-last-year . I'll email with more details offline.